live-examples/css-examples/margin.html

```diff
@@ -0,0 +1,47 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="margin">
```

Comment:
I am thinking that, if we know that the property is well supported given our browser support matrix, we do not have to add the data attribute.

Reply:
At the moment it seems that if this is omitted, the editor assumes that the property is not supported, and just displays the static example. So I have left this in.

Reply:
Oh that is certainly a bug. I will look into it.

Reply:
It should be fixed in https://github.com/mdn/interactive-examples/pull/381.